README.md

```diff
@@ -149,7 +149,7 @@ TopoJSON always consists of a single topology object. A topology may contain any
 
 A topology is a TopoJSON object where the type member's value is “Topology”.
 
-  * A topology must have a member with the name “objects” whose value is another object, each of which is a named geometry object.
+  * A topology must have a member with the name “objects” whose value is another object, whose each property is a geometry object.
```

Comment:
I guess the spec also has: https://github.com/mbostock/topojson-specification#215-objects

> 2.1.5. Objects
> A topology must have an “objects” member whose value is an object. This object may have any number of members, whose value must be a geometry object.

so it's not really _that_ ambiguous.